common/src/main/java/com/wynntils/core/webapi/profiles/item/ItemType.java

```diff
@@ -2,33 +2,44 @@
  * Copyright © Wynntils 2022.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.wc.objects.items;
+package com.wynntils.core.webapi.profiles.item;
 
 import com.google.gson.annotations.SerializedName;
+import com.wynntils.wc.objects.ClassType;
 
 public enum ItemType {
     @SerializedName("SPEAR")
-    Spear,
```

Comment:
If you're cleaning this up anyway, I liked what you did with the DamageType enum. Imho enums should follow the UPPERCASE convention, and it's much better to fix `toString()`.